pre-receive-hooks/block_skip_checks.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+#
+# Check and reject commits with "skip-checks: true" trailer lines.
+# 
+# This hook basically disables the following feature:
+# https://help.github.com/articles/about-status-checks/#skipping-and-requesting-checks-for-individual-commits
+
+ERROR_MSG="[POLICY] Skipping checks is not allowed. Please remove trailer lines with \"skip-checks: true\"."
+
+while read OLDREV NEWREV REFNAME ; do
+  for COMMIT in `git rev-list $OLDREV..$NEWREV`;
```

Comment:
I think this call would fail if you push a new branch as `$OLDREV` is 40 zeros in that case. See https://github.com/github/platform-samples/blob/master/pre-receive-hooks/block_confidentials.sh#L38